lib/inets/doc/src/httpd_util.xml

```diff
@@ -264,7 +264,7 @@
       <name since="">reason_phrase(StatusCode) -> Description</name>
       <fsummary>Returns the description of an HTTP 1.1 status code.</fsummary>
       <type>
-        <v>StatusCode = 100| 200 | 201 | 202 | 204 | 205 | 206 | 300 | 301 | 302 | 303 | 304 | 400 | 401 | 402 | 403 | 404 | 405 | 406 | 410 411 | 412 | 413 | 414 415 | 416 | 417 | 500 | 501 | 502 | 503 | 504 | 505</v>
+        <v>StatusCode = 100| 200 | 201 | 202 | 204 | 205 | 206 | 300 | 301 | 302 | 303 | 304 | 308 | 400 | 401 | 402 | 403 | 404 | 405 | 406 | 410 411 | 412 | 413 | 414 415 | 416 | 417 | 500 | 501 | 502 | 503 | 504 | 505</v>
```

Comment:
While editing this line. Could you maybe fix formatting details ?
1. add space after `100`
2. add | after `410`
3. add | after `414`

Reply:
I even splitted the line. Is it OK?

Reply:
thanks